From 3391fa810ca323203b036a68af7f3c5802280918 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Klime=C5=A1?= <jklimes@redhat.com>
Date: Wed, 22 Jan 2014 10:03:44 +0100
Subject: [PATCH] libnm-glib: export missing get_gateway() and get_searches()
 functions
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

They were added by db9b7e10aca5456ec4960b75617e032209b98bc1 commit.

Signed-off-by: Jiří Klimeš <jklimes@redhat.com>
---
 libnm-glib/libnm-glib.ver | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libnm-glib/libnm-glib.ver b/libnm-glib/libnm-glib.ver
index cb7b599..bf69db4 100644
--- a/libnm-glib/libnm-glib.ver
+++ b/libnm-glib/libnm-glib.ver
@@ -215,8 +215,10 @@ global:
 	nm_dhcp6_config_new;
 	nm_ip4_config_get_addresses;
 	nm_ip4_config_get_domains;
+	nm_ip4_config_get_gateway;
 	nm_ip4_config_get_nameservers;
 	nm_ip4_config_get_routes;
+	nm_ip4_config_get_searches;
 	nm_ip4_config_get_type;
 	nm_ip4_config_get_wins_servers;
 	nm_ip4_config_new;
@@ -224,8 +226,10 @@ global:
 	nm_ip6_address_object_array_get_type;
 	nm_ip6_config_get_addresses;
 	nm_ip6_config_get_domains;
+	nm_ip6_config_get_gateway;
 	nm_ip6_config_get_nameservers;
 	nm_ip6_config_get_routes;
+	nm_ip6_config_get_searches;
 	nm_ip6_config_get_type;
 	nm_ip6_config_new;
 	nm_ip6_route_object_array_get_type;
-- 
1.7.11.7

